Sources/SwiftIfConfig/BuildConfiguration.swift

```diff
@@ -214,6 +214,21 @@ public protocol BuildConfiguration {
   /// pointer authentication scheme.
   func isActiveTargetPointerAuthentication(name: String) throws -> Bool
 
+  /// Determine whether the given name is the active target object file format (e.g., ELF).
+  ///
+  /// The target object file format can only be queried by an experimental 
+  /// syntax `_objectFileFormat(<name>)`, e.g.,
+  ///
+  /// ```swift
+  /// #if _objectFileFormat(ELF)
+  /// // Special logic for ELF object file formats
+  /// #endif
+  /// ```
+  /// - Parameters:
+  ///   - name: The name of the object file format.
+  /// - Returns: Whether the target object file format matches the given name.
+  func isActiveTargetObjectFileFormat(name: String) throws -> Bool
```

Comment:
Could you add a default implementation returning `false` so that we can land this `swift-syntax` PR independently before swift-syntax counterpart?

Reply:
Added!

Reply:
The test failure is because this default implementation is not `public` 🙏

Reply:
We should also make this `@_spi(ExperimentalLanguageFeatures)` so that clients can’t call it as public API and start relying on it.